packages/client/src/__tests__/integration/integration-runner.test.ts

```diff
@@ -0,0 +1,48 @@
+import * as child_process from 'child_process'
+import * as fs from 'fs'
+import * as path from 'path'
+import * as util from 'util'
+
+const exec = util.promisify(child_process.exec)
+
+const MAX = 2
+const BACKOFF_MS = 100
+const excluded = ['exhaustive-schema', 'exhaustive-schema-mongo', 'client-engine', 'ignore']
```

Comment:
These ones were taking a very long time doing it this way and so I have skipped them for now until I can investigate more.